### ISSUE TITLE
Adding info about reverse sorting to the doc

### ISSUE THREAD

**Issue by MrModest** [gristlabs/grist-help#298]
In the doc there's no info on how to do reverse sorting. Only after searching through the source code I figured out that I need to add minus (`-`) before field name. It would be great to have this info in the doc, so other people won't have to go to the source for this simple info.

We need to add this info in these two places:
- https://github.com/gristlabs/grist-help/blob/5a8e160900969caea26c34efac2610e20dc1199c/help/functions.md?plain=1#L158
- https://github.com/gristlabs/grist-help/blob/5a8e160900969caea26c34efac2610e20dc1199c/help/formulas.md?plain=1#L283

**Comment by dsagal** [comment]
Good point. Another place where it's very helpful to be aware of is for `lookupOne`. Since `lookupOne` returns the first of matching records, something like `sort_by="-Date"` can be used to find the latest of matching records.
